docs/src/ref/build-and-create.md

```diff
@@ -0,0 +1,39 @@
+# skip_create, to_create, and initialize_with
+
+The `skip_create`, `to_create`, and `initialize_with` methods control how
+factory\_bot interacts with the [build strategies](build-strategies.html).
+
+These methods can be called within a `factory` definition block, to scope their
+effects to just that factory; or within `FactoryBot.define`, to effect global
+change.
+
+## initialize_with
+
+The `initialize_with` method takes a block and returns an instance of the
+factory's class. It has access to the `attributes` method, which is a hash of
+all the fields and values for the object.
+
+The default definition is:
+
+```ruby
+initialize_with { new }
```

Comment:
I believe that an `attributes` method is available within this block. Is the default block value `new(attributes)`?

Reply:
FB new-s up a blank object and then assigns to it using setters. This is perfect for AR, and bad for most other objects.